### DISCUSSION TITLE
Regarding grading policy in this course

### DISCUSSION THREAD

**Discussion by Yassawe** [seominjoon/kaist-ai605#154] [Q&A] [answered]
Hello!

Sorry if that has been stated before, I wanted to ask whether the grading in this course is relative or fixed?

Let's say one student takes 100 and the other student takes 114. In this case, they both get A+, or only the second student gets A+ and everyone else is graded relative to him?

Thanks!

**Comment by seominjoon** [comment, marked as answer]
All grades up to A are absolute grading. Please see the course syllabus on https://seominjoon.github.io/kaist-ai605/
A+ is relative though. At most 10% of the students get A+.
